### PR TITLE
Fix hyperlinks and add a limitation. 

### PR DESCRIPTION
**Pull request by mcchampions**
Update links, although "Camotoy/GeyserHacks" still works.

### CODE REVIEW

**Review by onebeastchris — CHANGES_REQUESTED** [review]
lol you should've been a reviewer when the api docs were PR'd, causing most of these wrong links :p
Thanks!

File: _docs/geyser/current-limitations.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ The following things cannot be fixed without changes to Bedrock or the Java prot
 - Various command arguments for any command that doesn't use the Minecraft Brigadier library
 - Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
 - Unable to see banner layers past 6
-- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: https://github.com/Camotoy/GeyserHacks
+- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](https://github.com/GeyserMC/Hurricane)
```
[the remark this answers is not in the document]
Comment (mcchampions) [reply _docs/geyser/current-limitations.md:20]: You're right, I'll make changes。

**Review by onebeastchris — APPROVED** [review]
Looks good to me 👍
I'll leave the PR open for a bit though, incase you find more dead links.

File: _data/providers.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "description_templates": {
-    "default": "Get Geyser as a plugin, as in the [setup guide](https://wiki.geysermc.org/geyser/setup/). Use the same port as your Java server for the Bedrock port in your config (either by setting it yourself, or enabling \"clone-remote-port\") and connect with the same IP and port as you would on Java."
+    "default": "Get Geyser as a plugin, as in the [setup guide](/geyser/setup/). Use the same port as your Java server for the Bedrock port in your config (either by setting it yourself, or enabling \"clone-remote-port\") and connect with the same IP and port as you would on Java."
```
Comment (onebeastchris) [inline _data/providers.json:3]: Please revert this, this is done on purpose. For example. the Geyser discord bot depends on this file to show up to date provider instructions, so any relative link wouldn't work.

Also, there's another PR on the wiki that's changing the providers.json quite drastically, so I would avoid changes to this file that go beyond updating links/fixing typos

File: _data/toc.yml [hunk 3]
```diff
@@ -60,6 +60,8 @@
           url: "geyser/extensions/#creating-commands-with-geyser-extensions"
     - title: Global API
       url: "geyser/global-api/"
+    - title: Geyser and Secure ChatGeyser and Secure Chat
```
Comment (onebeastchris) [inline _data/toc.yml:63]: This page wasn't added to the TOC on purpose, instead, it's linked to in e.g. common issues

File: _docs/geyser/current-limitations.md [hunk 4]
```diff
@@ -17,7 +17,8 @@ The following things cannot be fixed without changes to Bedrock or the Java prot
 - Various command arguments for any command that doesn't use the Minecraft Brigadier library
 - Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
 - Unable to see banner layers past 6
-- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: https://github.com/Camotoy/GeyserHacks
+- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](/other/hurricaneh)
+- Can't move mounted entities with carrot/warped fungus on a stick(1.17.x & above). Hurricane also works around this.
```
Comment (onebeastchris) [inline _docs/geyser/current-limitations.md:21]: I'd remove the version here, and instead add e.g. camels to this list - Camels specifically are not fixed by Hurricane though.

File: _docs/other/community-geyser-projects.md [hunk 5]
```diff
@@ -75,7 +75,7 @@ Plugin that blocks Java players from joining your Geyser server, so you can have
 
 Creator: [Camotoy](https://github.com/Camotoy)
 
-## GeyserPlaceholders
+## ~~GeyserPlaceholders~~(Discontinued)
```
Comment (onebeastchris) [inline _docs/other/community-geyser-projects.md:78]: I would remove this from the list if it's discontinued

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: _docs/other/community-geyser-projects.md (hunk 5 above)
Comment (mcchampions) [reply _docs/other/community-geyser-projects.md]: ok

File: _docs/geyser/current-limitations.md [hunk 6]
```diff
@@ -17,12 +17,14 @@ The following things cannot be fixed without changes to Bedrock or the Java prot
 - Various command arguments for any command that doesn't use the Minecraft Brigadier library
 - Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
 - Unable to see banner layers past 6
-- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: https://github.com/Camotoy/GeyserHacks
+- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](/other/hurricaneh)
```
Comment (onebeastchris) [inline _docs/geyser/current-limitations.md:20]:
```suggestion
- Movement issues around bamboo due to offset differences between Java and Bedrock. [Hurricane](/other/hurricane) can work around it.
```

The warning itself is on the page, plus there was a typo

File: _includes/sidebar-page.html [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <a class="d-block text-decoration-none" href="{{ site.repo }}/edit/{% if site.github_branch %}{{ site.github_branch }}{% else %}master{% endif %}/{{ page.path }}" target="_blank"><i class="bi bi-pencil"></i> Edit this page</a>
 <a class="d-block text-decoration-none" href="{{ site.repo }}/issues/new?labels={% if page.editable %}{{ page.editable }}{% else %}question{% endif %}&title=Question:&body=Question on: {{ page.path }}" target="_blank"><i class="bi bi-github"></i> Create Wiki issue/bug report</a>
-<a class="d-block text-decoration-none" href="{{ page.repo }}/issues/new" target="_blank"><i class="bi bi-list-check"></i> Create Geyser issue/bug report</a>
+<a class="d-block text-decoration-none" href="https://github.com/GeyserMC/Geyser/issues/new/choose" target="_blank"><i class="bi bi-list-check"></i> Create Geyser issue/bug report</a>
```
Comment (onebeastchris) [inline _includes/sidebar-page.html:3]: Please revert this, there's a reason we set these links using a variable (so we don't have to update them across all usages)

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: _includes/sidebar-page.html (hunk 7 above)
Comment (mcchampions) [reply _includes/sidebar-page.html]: ok

**Review by onebeastchris — APPROVED** [review]
Looks good now 👍

**Review by rtm516 — APPROVED** [review]

**Pull request merged**